analyzers/src/SonarAnalyzer.CSharp/Rules/EmptyMethodCodeFix.cs

```diff
@@ -111,24 +106,19 @@ private static async Task RegisterCodeFixesForMethodsAsync(CodeFixContext contex
                                                 SyntaxFactory.ArgumentList(),
                                                 null))
                                     }))
-                                    .WithTriviaFrom(method.Body)
+                                    .WithTriviaFrom(methodBody)
                                     .WithAdditionalAnnotations(Formatter.Annotation));
                         return Task.FromResult(context.Document.WithSyntaxRoot(newRoot));
                     },
                     TitleThrow),
                 context.Diagnostics);
         }
 
-        private static bool NamespaceNeedsToBeAdded(BaseMethodDeclarationSyntax method,
-            SemanticModel semanticModel)
-        {
-            return !semanticModel.LookupNamespacesAndTypes(method.Body.CloseBraceToken.SpanStart)
+        private static bool NamespaceNeedsToBeAdded(BlockSyntax methodBody, SemanticModel semanticModel) =>
+            !semanticModel.LookupNamespacesAndTypes(methodBody.CloseBraceToken.SpanStart)
                 .OfType<INamedTypeSymbol>()
-                .Any(nt =>
-                    nt.IsType &&
-                    nt.Name == LiteralNotSupportedException &&
-                    nt.ContainingNamespace.Name == LiteralSystem);
-        }
+                .Any(nt => nt.IsType
```

Comment:
you can also rename `nt` to `symbol` since we don't use abbreviations.

Reply:
Thanks for pointing this out! I missed it completely.
According to our coding style: Single variable lambdas should use x as the variable name (based on lambda calculus λx)
So, I went for x.